src/router.js

```diff
@@ -172,7 +172,7 @@ export class Router extends Resolver {
     const baseHref = baseElement && baseElement.getAttribute('href');
     super([], Object.assign({
       // Default options
-      baseUrl: baseHref && Resolver.__createUrl(baseHref, document.URL).href.replace(/[^\/]*$/, '')
```

Comment:
This was an unnecessary escape (complained by linter rule). Most characters don't need to be escaped inside a character group `[]`.